cap/jsonschemas/deposits/records/lhcb-v1.0.0.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Deposit schema.",
+  "description": "Describe information needed for deposit module.",
+  "properties": {
+    "basic_info": {
+      "type": "object",
+      "properties": {
+        "anal_name": {
+          "type": "string",
+          "title": "Analysis Name"
+        }
+      }
+    },
+    "$schema": {
+      "type": "string"
+    },
+    "_deposit": {
```

Comment:
Yes, will come later.. We need first to fix the dereference of the subschemas ("alOf"s, etc) ..either with resolvers or a script to bundle them all